src/qibolab/tests/test_instruments_qmsim.py

```diff
@@ -77,8 +78,9 @@ def plot():
             if hasattr(samples, con):
                 sample = getattr(samples, con)
                 group = file.create_group(con)
-                for port, waveforms in sample.analog.items():
-                    group[port] = waveforms
+                for port, waveform in sample.analog.items():
+                    dset = group.create_dataset(port, (SIMULATION_DURATION,), compression="gzip")
+                    dset[:] = waveform
```

Comment:
Why is this not used anywhere else? What do you need the `dset` variable for?

Reply:
This was used to save the `waveform` array to the hdf5 file on disk, but I have now simplified to avoid creating the `dset` variable.